src/scss/_variables.scss

```diff
@@ -4,7 +4,8 @@ $ph-border-radius:        2px !default;
 
 $ph-gutter:               30px !default;
 $ph-spacer:               15px !default;
+$ph-item-border:          1px solid darken($ph-bg, 10%) !default;
```

Comment:
I think we should rename it to `$ph-border` and move it on line 4, to match the others. What do you think?